src/main/resources/assets/api-docs.yaml

```
@@ -3437,6 +3437,35 @@ paths:
           description: User not found
         500:
           description: Server error.
+  /api/user/role/{roleName}:
+    get:
+      summary: Get a list of users that you have access to based on your role
+      description: Finds users based on the user role and permissions
+      parameters:
+        - name: roleName
+          in: path
+          description: The name of the role of the user
+          schema:
+            type: String
+            nullable: false
```

This will fix the swagger bug you were seeing:
```suggestion
            type: string
            required: true
```